packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -37,8 +75,26 @@ const propTypes = {
 
 class EmbeddedContentConsumer extends React.Component {
   componentDidMount() {
+    // Merging the iframe options props
+    const frameOptions = { ...this.props.options };
+
+    if (this.props.srcdoc) {
+      if (frameOptions.iframeAttrs === null
+        || frameOptions.iframeAttrs === undefined) {
+        frameOptions.iframeAttrs = {};
+      }
+
+      Object.assign(frameOptions.iframeAttrs, { srcdoc: this.props.srcdoc });
+      Object.keys(frameOptions.iframeAttrs).forEach(key => {
```

Comment:
Question: Why are we filtering out null-or-undefined keys? Theoretically they shouldn't matter to the outcome if they don't have real values should they?

Reply:
My initial thought was that we don't want to set bad values for the keys. However, I think by not filtering out the null or undefined, that would give the consumers more flexibility if they need to set something to null or undefined.

I will go ahead and remove this logic.